### DISCUSSION TITLE
Request: "Join Squad" button redirects to squad page from "Find squads" page

### DISCUSSION THREAD

**Discussion by gouldcs** [dailydotdev/daily#1583] [Feature Requests]
### What went wrong? 🤔

When exploring squads in the "find squads" page, it would be nice if I could opt to join a squad without it redirecting to the squad page. I'd like to be able to click "join squad" and keep exploring the other squads on the page, similar to how other platforms function.

### Expected Behavior

When clicking "Join Squad", don't redirect to the squad page. Stay on the "Find Squads" page.

### Steps to Reproduce Issue

```bash
1. Go to 'Find Squads' page
2. Click on 'Join Squad'
3. Observe redirect
```

### Solution Proposed

Don't redirect on click, just update the button to show you've joined successfully.

### Environment

Chrome 129.0.6668.91

### Browsers

Chrome

### OS

Mac

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by rebelchris** [comment]
Hi @gouldcs,

Thanks for the suggestion I've raised it internally with our product team.
Will move this to a feature request, as it's not really a bug.
